### PR TITLE
doc: Generates v1.15.3

### PR DESCRIPTION
**Pull request by oarbusi**
## Description

generate changelog for v1.15.3

Link to any related issue(s): CLOUDP-239771

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [ ] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code
- [ ] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [ ] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,70 @@
 # Changelog
 
+## [v1.15.3](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.15.3) (2024-03-27)
+
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v0.1.0-test...v1.15.3)
+
+**Enhancements**
+
+- feat: Allows user to specify to use an existing tag for release [\#2053](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2053) ([maastha](https://github.com/maastha))
```
Comment (AgustinBettati) [inline CHANGELOG.md]: this is not customer facing, would move to internal improvement and rename to `chore:`. With this in mind would remove enhancements section

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,70 @@
 # Changelog
 
+## [v1.15.3](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.15.3) (2024-03-27)
+
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v0.1.0-test...v1.15.3)
```
Comment (AgustinBettati) [inline CHANGELOG.md]:
```suggestion
[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.15.2...v1.15.3)
```

File: CHANGELOG.md (hunk 1 above)
Comment (oarbusi) [reply CHANGELOG.md]: Done

**Review by EspenAlbert — APPROVED** [review]
LGTM!

**Review by carriecwk — APPROVED** [review]
LGTM!

**Pull request merged**